src/test/java/seedu/internhunter/model/ModelManagerTest.java

```diff
@@ -237,58 +237,58 @@ public void getFilteredProfileListSize_returnsEqual() {
+        assertEquals(companyItemList, modelManager.getCompanyItemList());
     }
 
     @Test
     public void getProfileItemList_testIfEqualProfileList_returnsEqual() {
         modelManager.addProfileItem(profileItemBuilder.build());
-        assertEquals(modelManager.getProfileItemList(), profileItemList);
+        assertEquals(profileItemList, modelManager.getProfileItemList());
     }
 
     @Test
     public void deleteCompany_deleteCompanyFromList_returnsEqual() {
         modelManager.addCompany(companyItemBuilder.build());
         modelManager.deleteCompany(companyItemBuilder.build());
-        assertEquals(modelManager.getFilteredCompanyList(), emptyCompanyItemList);
+        assertEquals(emptyCompanyItemList, modelManager.getFilteredCompanyList());
     }
 
     @Test
     public void deleteApplication_deleteApplicationFromList_returnsEqual() {
         modelManager.addApplication(applicationItemBuilder.build());
         modelManager.deleteApplication(applicationItemBuilder.build());
-        assertEquals(modelManager.getFilteredApplicationList(), emptyApplicationItemList);
+        assertEquals(emptyApplicationItemList, modelManager.getFilteredApplicationList());
     }
 
     @Test
     public void deleteProfileItem_deleteProfileItemFromList_returnsEqual() {
         modelManager.addProfileItem(profileItemBuilder.build());
         modelManager.deleteProfileItem(profileItemBuilder.build());
-        assertEquals(modelManager.getFilteredProfileList(), emptyProfileItemList);
+        assertEquals(emptyProfileItemList, modelManager.getFilteredProfileList());
     }
 
     @Test
     public void getUnfilteredCompanyList_equals_true() {
         modelManager.addCompany(companyItemBuilder.build());
         ItemList<CompanyItem> companyItemItemList = new ItemList<>();
         companyItemItemList.addItem(companyItemBuilder.build());
-        assertEquals(modelManager.getUnfilteredCompanyList(), companyItemItemList);
+        assertEquals(companyItemItemList, modelManager.getUnfilteredCompanyList());
     }
 
     @Test
     public void getUnfilteredProfileList_equals_true() {
         modelManager.addProfileItem(profileItemBuilder.build());
         ItemList<ProfileItem> profileItemItemList = new ItemList<>();
         profileItemItemList.addItem(profileItemBuilder.build());
-        assertEquals(modelManager.getUnfilteredProfileList(), profileItemItemList);
+        assertEquals(profileItemItemList, modelManager.getUnfilteredProfileList());
     }
 
     @Test
     public void getUnfilteredApplicationList_equals_true() {
         modelManager.addApplication(applicationItemBuilder.build());
         ItemList<ApplicationItem> applicationItemItemList = new ItemList<>();
         applicationItemItemList.addItem(applicationItemBuilder.build());
-        assertEquals(modelManager.getFilteredApplicationList(), applicationItemList);
+        assertEquals(applicationItemList, modelManager.getFilteredApplicationList());
```

Comment:
Should this be using `getUnfilteredApplicaitonList()` instead?

Reply:
good catch resolved